doc/classes/Physics2DDirectSpaceState.xml

```diff
@@ -17,8 +17,8 @@
 			<argument index="0" name="shape" type="Physics2DShapeQueryParameters">
 			</argument>
 			<description>
-				Check whether the shape can travel to a point. If it can, the method will return an array with two floats: The first is the distance the shape can move in that direction without colliding, and the second is the distance at which it will collide.
-				If the shape can not move, the array will be empty.
+				Check how far the shape can travel toward a point. Note that both the shape and the motion are supplied through a [Physics2DShapeQueryParameters] object. The method will return an array with two floats between 0 and 1: The first is how far the shape can move toward the point without triggering a collision, and the second is the distance at which a collision will occur. If no collision is detected, the returned array will be [1, 1].
```

Comment:
- Method descriptions should use the present tense: "Checks"

> and 1: The

the : should be a period.

> second is the distance 

This is misleading. The second parameter is not a distance, but the *fraction* of the distance. That's why a return of `1` indicates the body moved the full distance.